src/Engine/GraphicsHandler/I_GraphicsHandler.h

```diff
@@ -8,19 +8,191 @@
 class I_GraphicsHandler {
 public:
     virtual ~I_GraphicsHandler() {}
+    /**
+     * Used to intiate the GraphicsHandler
+     * Loads the vector of <code>graphic_paths</code> which are file paths
+     * to <code>GraphicsHandler::game_graphics_</code>, which is a map of std::string to SDL_Textures*
+     * @param path is the relative (recommended) or absolute path to graphics assets that will 
+     *  be loaded when init is called. This initial load avoids having to load assets while the
+     *  level has started, and can be done when "loading" the level
+     * @return returns <code>void</code>. However, <code>GraphicsHandler::internal_load_image()</code> calls 
```

Comment:
the @return I think can be confusing, since you'er right it is void. It'd be better to put this stuff in the note.